br/pkg/lightning/mydump/loader.go

```diff
@@ -731,6 +732,14 @@ func calculateFileBytes(ctx context.Context,
 
 // SampleFileCompressRatio samples the compress ratio of the compressed file.
 func SampleFileCompressRatio(ctx context.Context, fileMeta SourceFileMeta, store storage.ExternalStorage) (float64, error) {
+	failpoint.Inject("SampleFileCompressRatio", func(val failpoint.Value) {
```

Comment:
```suggestion
	failpoint.Inject("SampleFileCompressPercentage", func(val failpoint.Value) {
```

because we use `float64(v)/100` below